### PR TITLE
Adding ACFT docker context

### PR DESCRIPTION
**Pull request by skanakamedal**

### CODE REVIEW

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/asset.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu
```
Comment (elesel) [inline acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/asset.yaml:1]: What's the purpose of this environment? The name doesn't adhere to [naming conventions](https://github.com/Azure/azureml-assets/wiki/Asset-naming-convention#environments), but if it's largely being used to run finetuning components we can deviate a bit.

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#PTCA image
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:biweekly.202301.2
```
Comment (elesel) [inline acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile:2]: Use template tags to keep this environment up to date: https://github.com/Azure/azureml-assets/wiki/Environments#template-tags

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+RUN pip uninstall -y onnxruntime-training
+RUN pip install -i https://test.pypi.org/simple/ onnxruntime-training==1.14.0.dev20230119001
+
+# ENV NCCL_DEBUG=INFO
+ 
+# ls root dir
+RUN ls -lh
+
```
Comment (elesel) [inline acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile:10]: Please remove these.
```suggestion
```

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+USER root
+RUN apt-get -y update
+
+# pip install requirements
+RUN pip install https://automlcesdkdataresources.blob.core.windows.net/finetuning-models/prp_wheels/azureml_acft_accelerator-0.1.0.0-py3-none-any.whl
```
Comment (elesel) [inline acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile:15]: Please get the azureml_acft_accelerator and onnxruntime-training packages into regular PyPI.

References to wheels in storage blobs shouldn't be in azureml-assets.

**Review by elesel — CHANGES_REQUESTED** [review]

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile (hunk 2 above)
Comment (savitamittal1) [reply acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile:2]: why are we using AI infra as base image? Shouldn't we be using ACPT curated env?

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile (hunk 4 above)
Comment (savitamittal1) [reply acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile:15]: +1

File: acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile (hunk 2 above)
Comment (vizhur) [reply acft/environments/acft-ubuntu20.04-py38-pytorch1.13.1-cuda11.7-gpu/context/Dockerfile:2]: @Savita, you should not use CE image as a base image for another environment

**Pull request closed without merging**